### PR TITLE
Added support for more interface types in IOSXR

### PR DESCRIPTION
**Pull request by m1009d**
The current regexp for `ETHERNET_INTERFACES` in the `cisco_iosxr/patterns.py` file will not catch `TenGigE` or `HundredGigE` interfaces. 
Please see my proposed simple change in the regexp.

### CODE REVIEW

**Comment by m1009d** [comment]
Hi @carlmontanari , 
I've added the `ethernet` interface type and pushed the change but I have no experience with `make lint` and I don't know exactly how should I use it.

**Comment by carlmontanari** [comment]
Nice! Thanks for the update!

Sorry about that -- [`make`](https://www.gnu.org/software/make/) is used for some "shortcut" type stuff in this case. `lint` is a directive that runs `black` and other formatters/linters -- you can see that [here](https://github.com/scrapli/scrapli_cfg/blob/main/Makefile#L1-L6).

TL;DR to get build passing is you just need to run `python -m black .` like the makefile does -- black will auto format the code and then should be good to go! HTH!

Carl

**Comment by carlmontanari** [comment]
Boom, LGTM, thank you much @m1009d
